CCPAConsentViewController/Assets/JSReceiver.js

```diff
@@ -42,31 +42,40 @@
   }
 
   var handleMessageEvent = function(SDK) {
-    return function(name, payload) {
-      switch(name) {
+    return function(eventData) {
+        debugger
```

Comment:
We should probably remove the `debugger` from the code.

Reply:
Removed the debugger from code